app/views/groups/new.html.erb

```diff
@@ -0,0 +1,5 @@
+<section class="main-section">
+<h1 class="header">NEW CATEGORY</h1>
+<%= render "form", group: @group %>
+<div class="links-wrapper"> <%= link_to "Back to Categories", groups_path, class:"link" %></div>
+</section>
```

Comment:
- [ ] Kindly improve the styling of the inputs on your category new page to follow the design. Right now they seem like basic HTML elements without any styling.

- [ ] Kindly also try to maintain the colors so the heading are visible like in the design.

  Your Implementation | Behance Design
  :---:|:---:
  ![kifr](https://user-images.githubusercontent.com/36542986/235439316-fb062e61-ff4f-457a-9ee1-71e4dbbf4fe1.jpg) | ![kisw](https://user-images.githubusercontent.com/36542986/235439206-4f18a4bb-fc69-4f62-8d98-efa6e41d185b.jpg)